src/Sulu/Bundle/MediaBundle/Resources/public/js/services/overlay-manager.js

```diff
@@ -12,6 +12,7 @@ define(function() {
     'use strict';
 
     var instance = null,
+        overlayOpened = false,
```

Comment:
@nnatter please use a non static variable as the overlay-manager is already a singleton